l10n_it_account/models/account_tax.py

```diff
@@ -158,22 +193,33 @@ def _compute_totals_tax(self, data):
         registry_type = data.get("registry_type", "customer")
         if data.get("journal_ids"):
             context["vat_registry_journal_ids"] = data["journal_ids"]
+            if data.get("rc_journal_ids"):
+                context["vat_registry_journal_ids"] += data["rc_journal_ids"]
 
         tax = self.env["account.tax"].with_context(**context).browse(self.id)
         tax_name = tax._get_tax_name()
         base_balance = tax.base_balance
         balance = tax.balance
         deductible_balance = tax.deductible_balance
         undeductible_balance = tax.undeductible_balance
+        debit_balance = tax.debit_balance
+        credit_balance = tax.credit_balance
         if registry_type == "supplier":
             base_balance = -base_balance
             balance = -balance
             deductible_balance = -deductible_balance
             undeductible_balance = -undeductible_balance
+            debit_balance = -debit_balance
+            credit_balance = -credit_balance
+        if registry_type == "customer" and tax.type_tax_use == "purchase":
+            # caso reverse charge in regsitro IVA vendite
```

Comment:
I commenti, come i messaggi dei commit, devono essere in inglese, vedi https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#33idioms:
> Use English variable names and write comments in English.

Reply:
modificato